src/client/src/components/abstract/button/index.css

```diff
@@ -0,0 +1,26 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
```

Comment:
This import is already exist in index.css file in my branch.
And when it be merged the font will be on all css files.